controller/controller_manager.go

```diff
@@ -46,6 +46,9 @@ func StartControllers(logger logrus.FieldLogger, stopCh chan struct{}, controlle
 		return nil, nil, errors.Wrap(err, "unable to get client config")
 	}
 
+	config.Burst = 100
+	config.QPS = 50
```

Comment:
Curious. How to determine Burst and QPS?

Reply:
Another question: do we need to make them configurable?

Reply:
Same question. Are these 2 numbers large enough to satisfy the requests for the thousands of snapshot/backup CRs?

Reply:
The default value for those variable are 10 and 5: [link](https://github.com/kubernetes/client-go/blob/77f63643f951f19681397a995fe0916d2d5cb992/rest/config.go#L45-L46)  and [link](https://github.com/kubernetes/client-go/blob/77f63643f951f19681397a995fe0916d2d5cb992/rest/config.go#L353-L365). This was the reason for the backup and long upgrading issue in v1.2.4

I increased them to 10x. It is tested with 1k volume and 20k snapshot. Recurring job taking/deleting 1000 snapshots every minute. It was smooth. From monitoring, the actual QPS was about 16 per second (smaller than our max 50 QPS)

I don't think we need to make them configurable. Many projects hard coded it:
1.  Prometheus set the values to 100 and 100 [link](
https://github.com/prometheus-operator/prometheus-operator/blob/9d68ecf289d711c66bef39d2f83429265abc6986/pkg/k8sutil/k8sutil.go#L96-L97)
1. Rancher is doing 10k for discovering phase [link](https://github.com/rancher/support-bundle-kit/pull/23/files)
1. ingress-nginx is doing 1M which is basically no throttling on client side (AKA: beating the API server until it yells back): [link](https://github.com/kubernetes/ingress-nginx/blob/e4d53786e771cc6bdd55f180674b79f5b692e552/pkg/ingress/controller/launch.go#L252-L259)
1. Kubectl is 300 burst and 50 QPS